docs/extend/modular-resources/create.md

```diff
@@ -0,0 +1,528 @@
+        pass
+
+    # spin: unimplemented
+    async def spin(self, angle: float, velocity: float, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None, **kwargs):
+        pass
+
+    # set_velocity: unimplemented
+    async def set_velocity( self, linear: Vector3, angular: Vector3, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None, **kwargs):
+        pass
+
+    # set_power: set the linear and angular velocity of the left and right motors on the base
+    async def set_power(self, linear: Vector3, angular: Vector3, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None, **kwargs):
+
+        # stop the base if absolute value of linear and angular velocity is less than .01
+        if abs(linear.y) < 0.01 and abs(angular.z) < 0.01:
+            return self.stop(extra=extra, timeout=timeout)
+
+        # use linear and angular velocity to calculate percentage of max power to pass to SetPower for left & right motors
+        sum = abs(linear.y) + abs(angular.z)
+
+        self.left.set_power(power=((linear.y - angular.z) / sum), extra=extra, timeout=timeout)
+        self.right.set_power(power=((linear.y + angular.z) / sum), extra=extra, timeout=timeout)
+
+    # stop: stop the base from moving by stopping both motors
+    async def stop(self, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None, **kwargs):
+        self.left.stop(extra=extra, timeout=timeout)
+        self.right.stop(extra=extra, timeout=timeout)
+
+    # is_moving: check if either motor on the base is moving with motors' is_powered
+    async def is_moving(self, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None, **kwargs) -> bool:
+        return self.left.is_powered(extra=extra, timeout=timeout)[0] or self.right.is_powered(extra=extra, timeout=timeout)[0]
+```
+
+<br>
+</details>
+
+<file>__init__.py</file> registers the mybase custom model and API helper functions with the SDK.
+
+<details>
+  <summary>Click to view sample code from <file>__init__.py</file></summary>
+
+``` python {class="line-numbers linkable-line-numbers"}
+"""
+This file registers the MyBase model with the Python SDK.
+"""
+
+from viam.components.motor import *
+from viam.components.base import Base
+from viam.resource.registry import Registry, ResourceCreatorRegistration
+
+from .my_base import MyBase
+
+Registry.register_resource_creator(Base.SUBTYPE, MyBase.MODEL, ResourceCreatorRegistration(MyBase.new_base, MyBase.validate_config))
+```
+
+</details>
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Code a main program that serves as the entry point file to the module
+
+{{< tabs name="Sample SDK Main Program Code">}}
+{{% tab name="Go"%}}
+
+<file>main.go</file> is the demo Go module's entry point file.
+When executed, it initializes the mybase custom model and API helper functions from the registry.
```

Comment:
```suggestion
When executed, it initializes the `mybase` custom model.
```
I think the way the custom model uses the API *is* the custom model, so I think this can be shortened.